client/src/test/java/org/gluu/oxtrust/api/configuration/OxAuthConfigTest.java

```diff
@@ -0,0 +1,18 @@
+package org.gluu.oxtrust.api.configuration;
+
+import org.junit.Test;
+
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class OxAuthConfigTest {
+
+    @Test
+    public void url_validation_happy_path() {
+        String opPolicyUri = "http://ox.gluu.org/doku.php?id=oxauth:policy";
```

Comment:
we should move URL to test profile

Reply:
Done for all the other remarks as requested.
For the tests, this is only for local usage, Thomas is already working on an automating testing app.